pyvista/utilities/helpers.py

```diff
@@ -181,7 +182,8 @@ def parse_field_choice(field):
     return field
 
 
-def get_array(mesh, name, preference='cell', info=False, err=False):
+#TODO should be Union[Tuple, np.ndarray], but causes cascading static checking issues
+def get_array(mesh, name, preference='cell', info=False, err=False) -> Any:
```

Comment:
As explained in the `TODO` comment, this typing could be more precisely defined, but would require overhauling the entire hierarchy of called functions to behave correct. This works fine for now; addresses current issue while not being any worse than the previous typing implementation (which was none).